### DISCUSSION TITLE
Uncaught Error: Cannot find module '../geohash', with latest release?

### DISCUSSION THREAD

**Discussion by dtau00** [appy-one/acebase#138] [Q&A] [answered]
This just randomly started happening.  im loading the library from 

`<script src="https://cdn.jsdelivr.net/npm/acebase@latest/dist/browser.min.js"></script>`

without any code changes, suddenly getting the following error when loading library

`Uncaught Error: Cannot find module '../geohash'`

Is there a missing dependecy with the latest release?  Should i pull the lib from another source?  Tried install the lastet versions through npm and updating dependencies.

**Comment by appy-one** [comment, marked as answer]
Thanks for reporting this. Fix has been published in v1.22.1!

[![Spread the word](https://user-images.githubusercontent.com/26569719/169265089-3d593555-e1ad-4390-986b-877ac2c38a47.svg)](https://twitter.com/intent/tweet?button=&url=https://github.com/appy-one/acebase&text=I'm+using+@AcebaseRealtime+in+my+project+to+make+my+life+easier!&button=)  [![contribute](https://user-images.githubusercontent.com/26569719/169265318-30c4c6a5-7c89-46a0-a7a2-ef433a8192f4.svg)](https://github.com/appy-one/acebase#contributing)  [![Sponsor AceBase](https://user-images.githubusercontent.com/26569719/168233053-8e56b243-4140-40ab-9a30-4cb3cc149bfe.svg)](https://github.com/sponsors/appy-one)

**Reply by dtau00** [reply]
awesome as always!
